### DISCUSSION TITLE
How make template smaller like zoom out to 90%

### DISCUSSION THREAD

**Discussion by kashcode** [ng-matero/ng-matero#626] [General]
Is this related to density or which styles need change to achieve this.

**Comment by nzbin** [comment]
You can try `density: -2`  and use smaller font size `body: { font-size: 14px; }`.
https://material.angular.io/guide/theming#defining-a-theme

But it's not perfect. Maybe I should improve the whole theme.

**Reply by kashcode** [reply]
Why this is not perfect?

**Reply by nzbin** [reply]
v18 has released, now you can set `html { font-size: 14px }` to zoom in page

**Comment by ismailix** [comment]
body {
        zoom: 90%;
    }

you can try
